webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoRecentSpacesHamburgerNavigation.vue

```diff
@@ -1,74 +1,57 @@
 <template>
   <v-container class="recentDrawer" flat>
-    <v-flex class="filterSpaces">
+    <v-flex class="filterSpaces d-flex align-center">
+      <v-list-item-icon class="d-flex d-sm-none backToMenu me-2 my-5 icon-default-color" @click="closeMenu()">
+        <i class="fas fa-arrow-left"></i>
+      </v-list-item-icon>
+      <a  
+        v-if="canAddSpaces"
+        :href="allSpacesLink" 
+        class="addNewSpaceIcon px-2 primary rounded py-1">
+        <v-icon 
+          class="fas fa-plus white--text" 
+          size="16" />
+      </a>
```

Comment:
This should have been added with a Vuetify `v-btn`:
```html
<v-btn
  v-if="canAddSpaces"
  :href="allSpacesLink"
...
</v-btn>
```
This should have generated a link with `<a href...>` with a better style, effect and that offers the same UX as other buttons and parts in the product.